docs/HomeAssistantIntegration.md

```diff
@@ -0,0 +1,339 @@
+# HomeAssistant Integration
+
+## Introducton
+
+Whilst HomeAssistant provides an official component for Tesla vehicles, the component has not been updated recently, and does not have the sophistication of TeslaMate's polling mechanism, resulting in the component's default values keeping the vehicle awake and draining the battery.
+
+The ultimate goal of this guide is to consume as much of the TeslaMate polling data as possible to replace the majority of the official Tesla component's polling functionality. 
+
+If your intention is to only use read-only sensor values, those provided by TeslaMate via MQTT are sufficient, and you do not need to utilise the official Tesla compoent. If however you would like to be able to write values to the Tesla API (Lock/Unlock Doors or automate Climate), there is a solution which involves configuring an extremely high polling interval for the Tesla component and using automation to populate the values from the TeslaMate MQTT parameters.
+
+## Screenshot
+
+Coming soon
+
+## Current Status
+
+  * Sensors: All existing sensors available except for:
+    * Park Brake
+  * Locks: Not implemented
```

Comment:
There is Locket: yes on the teslamate dashboard (localhost:4000), so it must be available.